main.go

```diff
@@ -113,8 +116,12 @@ func main() {
 
 	if *redisMetricsOnly {
 		registry := prometheus.NewRegistry()
-		registry.Register(exp)
-		registry.Register(buildInfo)
+		if err = registry.Register(exp); err != nil {
+			log.Error(err)
```

Comment:
Let's just  `MustRegister` - if an error occurs (which it shouldn't) then continuing doesn't make much sense anyway but we should fail.